src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java

```diff
@@ -1,12 +1,7 @@
 package io.github.thebusybiscuit.slimefun4.implementation;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
No star imports please (very possible your IDE did this automatically)

Reply:
Yes, it did.